src/main/java/com/bwfcwalshy/flarebot/github/GithubListener.java

```diff
@@ -26,8 +26,9 @@ public void handle(PushEvent e) {
                 StringBuilder sb = new StringBuilder();
                 sb.append("```md\n");
                 int i = 1;
-                for (String file : commit.getAdded()) {
+                for (String filePath : commit.getAdded()) {
                     if (!(i++ >= 5)) {
```

Comment:
Change this to greater than also you can just do if(i <= 5) why complicate things?